src/wirecloud/defaulttheme/static/css/styledelements/styled_expander.scss

```diff
@@ -21,6 +21,7 @@
     width: 1em;
     text-align: center;
     display: inline-block;
+    font-family: 'FontAwesome';
```

Comment:
Was simply missing the font declaration for the icons to properly show up

Reply:
Hmm, this should be done using the FontAwesome standard way, by using the `fa` class on the toggle button.

Reply:
Okay, now I simply add the `fa` class to the same element